VMEncryption/main/handle.py

```diff
@@ -1858,6 +1906,11 @@ def enable_encryption_all_in_place(passphrase_file, encryption_marker, disk_util
                                                                                    status_prefix=status_prefix)
 
             if encryption_result_phase == CommonVariables.EncryptionPhaseDone:
```

Comment:
let's consider scenarios where encryption phase is disrupted and volume is partially encrypted. In this case, saving the settings might be good idea as we start?